appendices.tex

```diff
@@ -370,6 +370,146 @@ \chapter{Listing of application roles}
 \item [yearend\_run]
 \end{description}
 
+\chapter{Deprecated Content}
+\label{app-deprecated-content}
+
+\section{ledgersmb.conf}
+\label{app-deprecated-ledgersmb-conf}
+
+This ledgersmb.conf is deprecated as of 1 Jan 2023 and only remains here for historical use.
+
+The environment variable for controlling the configuration is \texttt{LSMB\_CONFIG\_FILE}.
+
+By default it pointed to \texttt{/etc/ledgersmb/ledgersmb.conf} and can be set to an alternate location.
+
+\subsubsection{[main] Section}
+\label{subsubsec-global-config-ledgersmb-conf-general}
+
+\begin{description}[style=nextline]
+	\item[language] Set language for login and admin pages. 
+	\item[date\_format] Set the date format for the admin pages.  The default is \texttt{yyyy-mm-dd}. Unset means detected from the browser)
+	\item[log\_level] Specifies the logging level to be used. This setting is ignored when the \texttt{log\_config} setting 
+	is supplied and indicates an existing file. Available values are OFF, FATAL, ERROR, WARN, INFO, DEBUG, TRACE.
+	\item [log\_config] Set the name of the extended logging configuration file. This file uses the \texttt{log4j/Log::Log4perl} syntax described at \url{https://metacpan.org/pod/Log::Log4perl#Configuration-files}
+	\item [dojo\_theme] This is the Dojo theme that is used by default when no other theme
+	has been selected.  The default if unset is \texttt{claro}. Other values can be \texttt{nihilo}, \texttt{soria}, and \texttt{tundra}.
+	
+	
+% From the old book, commented out for now.
+% 	\item[auth] @@@TODO
+% 	\item[tempdir] Directory to store temporary artifacts. E.g. PDF files are stored here before
+% 	being sent to the browser. The default on unix variants is /tmp/ledgersmb.
+% 	\item[cache\_templates] Determines if HTML templates used for the LedgerSMB UI will be stored
+% 	in precompiled form for improved performance. The value 0 (zero) means no caching; the value
+% 	1 means caching in the directory \$(tempdir)/lsmb\_templates; systems running multiple
+% 	LedgerSMB instances with customized UIs should use separate values for {\tt tempdir} for that
+% 	reason
+% 	\item[DBI\_TRACE] @@@TODO
+% 	\item[pathsep] @@@TODO
+% 	\item[latex] Identifies if the system has \LaTeX available: 0 means no \LaTeX, 1 means \LaTeX supported

```

Comment:
Doesn't apply since 1.4.

Reply:
Removed latex, although it was not clear to me if your comment applied to only the last item latex, or to all items?

Reply:
yep. it referred to 'latex'.